spec/support/connection_string_tests/valid-warnings.yml

```diff
@@ -11,41 +11,41 @@ tests:
                 port: ~
         auth: ~
         options: ~
-    # -
-    #     description: "Unsupported option values are ignored"
-    #     uri: "mongodb://example.com/?fsync=ifPossible"
-    #     valid: true
-    #     warning: true
-    #     hosts:
-    #         -
-    #             type: "hostname"
-    #             host: "example.com"
-    #             port: ~
-    #     auth: ~
-    #     options: ~
-    # -
-    #     description: "Repeated option keys"
-    #     uri: "mongodb://example.com/?replicaSet=test&replicaSet=test"
-    #     valid: true
-    #     warning: true
-    #     hosts:
-    #         -
-    #             type: "hostname"
-    #             host: "example.com"
-    #             port: ~
-    #     auth: ~
-    #     options:
-    #         replicaset: "test"
-    -
-        description: "Deprecated (or unknown) options are ignored if replacement exists"
-        uri: "mongodb://example.com/?wtimeout=5&wtimeoutMS=10"
-        valid: true
-        warning: true
-        hosts:
-            -
-                type: "hostname"
-                host: "example.com"
-                port: ~
-        auth: ~
-        options:
-            wtimeoutms: 10
+#    -
+#        description: "Unsupported option values are ignored"
+#        uri: "mongodb://example.com/?fsync=ifPossible"
+#        valid: true
+#        warning: true
+#        hosts:
+#            -
+#                type: "hostname"
+#                host: "example.com"
+#                port: ~
+#        auth: ~
+#        options: ~
+#    -
+#        description: "Repeated option keys"
+#        uri: "mongodb://example.com/?replicaSet=test&replicaSet=test"
+#        valid: true
+#        warning: true
+#        hosts:
+#            -
+#                type: "hostname"
+#                host: "example.com"
+#                port: ~
+#        auth: ~
+#        options:
+#            replicaset: "test"
+#    -
+#        description: "Deprecated (or unknown) options are ignored if replacement exists"
+#        uri: "mongodb://example.com/?wtimeout=5&wtimeoutMS=10"
```

Comment:
For the record, this is ignored because Ruby driver 2.x doesn't have any deprecated options to even trigger this.